docs/section19/19-4.md

```diff
@@ -0,0 +1,102 @@
+                - Get the get_most_used_weekday_hourly_schedule (this will be a list with 24 values for each hour of a day): `most_used_weekday_hourly_schedule = get_most_used_weekday_hourly_schedule(B_RMI, ASHRAE229,multiplier_sch)`  
+                - Reset y = 0: `y = 0`
+                - Check if each value in the design_cooling_multiplier_sch matches the corresponding hourly value in the get_most_used_weekday_hourly_schedule: `for x in list(design_cooling_multiplier_sch):`
+                    - Check if the hourly values are equal between the design_cooling_multiplier_sch and the get_most_used_weekday_hourly_schedule : `if most_used_weekday_hourly_schedule[y] != x: misc_pass_cooling = false`  
+                    - Add 1 to y: `y = y +1`  
+
+        - **Rule Assertion:** 
+        - Case 1: If lighting or ventilation space type is dwelling unit and all schedules pass per above then pass: `if space_lighting_space_type_is_dwelling_unit = true and all(inf_pass_cooling,occ_pass_cooling,int_lgt_pass_cooling,misc_pass_cooling) == true: outcome = "PASS"`  
+        - Case 2: elif the lighting or ventilation space type is NOT defined and (the building area type was not defined or the building area type is multifamily, dormitory, or hotel/motel) then outcome is UNDETERMINED: `elif space_lighting_space_type_is_defined == false and (bldg_area_is_defined == false or building_area_is_MF_dormitory_or_hotel == true): outcome = "UNDETERMINED" and raise_message "It is not clear from the RMD if <insert space.id> is a dwelling unit. If it is a dwelling unit it is required that it be modeled following the rule that for baseline cooling sizing runs in residential dwelling units, the infiltration, occupants, lighting, gas and electricity design day cooling schedules shalled be modeled using an equipment hourly schedule that is the same as the most used hourly weekday schedule from the annual simulation. This rule <insert "was" if all(inf_pass_cooling,occ_pass_cooling,int_lgt_pass_cooling,misc_pass_cooling) == true and insert "was not" if all(inf_pass_cooling,occ_pass_cooling,int_lgt_pass_cooling,misc_pass_cooling) == false> followed for this space if applicable."` ` 
+        - Case 3: elif lighting or ventilation space type is dwelling unit and not all schedules do not pass per above then Fail: `elif space_lighting_space_type_is_dwelling_unit = true and all(inf_pass_cooling,occ_pass_cooling,int_lgt_pass_cooling,misc_pass_cooling) == false: outcome = "FAIL" and raise_message "<insert space.id> does not appear to have been modeled following the rule that for baseline cooling sizing runs in residential dwelling units, the infiltration, occupants, lighting, gas and electricity design day cooling schedules shalled be modeled using an equipment hourly schedule that is the same as the most used hourly weekday schedule from the annual simulation."` 
```

Comment:
Optionally, it would be helpful to list schedules that are off (whether it's lighting, infiltration, occ, or misc. equipment.

Reply:
Done